### PR TITLE
_.unique is not a function

### PR DESCRIPTION
**Pull request by JamieMason**
Fixes;

```
$ npm install contributors -g
/usr/local/bin/contributors -> /usr/local/lib/node_modules/contributors/bin/contributors
contributors@0.5.0 /usr/local/lib/node_modules/contributors
├── progress@1.1.8
├── async@1.5.2
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── lodash@4.5.1
└── urllib@2.7.2 (media-typer@0.3.0, statuses@1.2.1, default-user-agent@0.0.1, any-promise@1.0.0, iconv-lite@0.4.13, debug@2.2.0, humanize-ms@1.0.1, digest-header@0.0.1)
```

```
$ contributors
/usr/local/lib/node_modules/contributors/lib/git/index.js:21
    var authors = _.unique(emails)
                    ^

TypeError: _.unique is not a function
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/contributors/lib/git/index.js:21:21)
    at ChildProcess.g (events.js:260:16)
    at emitTwo (events.js:92:20)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:818:16)
    at Socket.<anonymous> (internal/child_process.js:319:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```

### CODE REVIEW

**Comment by JamieMason** [comment]
Travis failed because "API rate limit exceeded"

File: package.json [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
-    "commander": ">=2.2.0",
-    "progress": ">=1.1.5",
-    "urllib": ">=0.5.13",
-    "lodash": ">=2.4.1",
-    "async": ">=0.9.0"
+    "async": "1.5.2",
+    "commander": "2.9.0",
+    "lodash.uniq": "4.2.0",
+    "progress": "1.1.8",
+    "urllib": "2.7.2"
```
Comment (fengmk2) [inline package.json]: `~` deps will be better. e.g.: `"urllib": "~2.7.2"`

Comment (JamieMason) [inline package.json]: No worries @fengmk2, whatever suits you best :+1: 

(as a background, I tend to use fixed version numbers [for this reason](https://github.com/JamieMason/shrinkpack#npm-shrinkwrap) but this is your project so it's totally up to you)

Comment (JamieMason) [inline package.json]: Hey actually, now I think of it, the `_.unique` bug happened for exactly this reason.

Comment (JamieMason) [inline package.json]: Added ~s to PR.

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
Released 0.5.1 to npm.

**Comment by xingrz** [comment]
thanks @fengmk2

**Pull request merged**